blog/_posts/2022-08-08-managing-tunnels-with-argocd.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: post
+title: Managing Inlets tunnels with Argo CD
+description: If you're managing multiple inlets tunnel servers, Argo CD could make your job simpler. We'll show you how to set up two inlets-pro TCP servers using the helm chart and SealedSecrets for the authentication tokens.
```

Comment:
> Meta descriptions can technically be any length, but Google generally truncates snippets to ~155-160 characters. It's best to keep meta descriptions long enough that they're sufficiently descriptive, so we recommend descriptions between 50 and 160 characters.

`wc` is showing 212 characters. Bring this down a little. You'll be able to use a longer snippet in the introduction or the first line of the actual post.

This field is mainly for SEO.